### PR TITLE
Add Twitter and FB OG image metadata

### PR DESCRIPTION
**Pull request by ganeshbsub**

### CODE REVIEW

File: _includes/head.html [hunk 1]
```diff
@@ -6,6 +6,8 @@
   <meta name="language" content="en"/>
   <meta name="content-language" content="en"/>
   <meta name="publisher" content="simplabs" />
+  <meta content="/images/logo@2x.png" property="og:image"/>
+  <meta name="twitter:image:src" content="/images/logo@2x.png">
```
Comment (marcoow) [inline _includes/head.html:10]: I guess these should be added by the seo gem? Are we using an old version maybe?

Comment (ganeshbsub) [reply _includes/head.html:10]: I checked the meta data on our site. Everything else(og: title, twitter: title, etc.) but the images are being generated. I tried this explicitly to see if this solved the issue.

**Comment by ganeshbsub** [comment]
Not sure why it still doesn't show an image. https://cards-dev.twitter.com/validator

**Comment by ganeshbsub** [comment]
[This](https://github.com/jekyll/jekyll-seo-tag/issues/84) might be the issue.

**Pull request merged**